docs/sync-overrides.md

```
@@ -0,0 +1,142 @@
+
+1. Open a PR in the downstream repo that includes:
+   - The new `.sync-overrides.yml` entry (skip_paths / substitutions, with `reason`).
+   - The corresponding file change (the actual divergence in the repo content).
+2. The reviewer flags any `reason` field that's missing or vague. "Required for this repo" is not a reason; "uses Cloud Run instead of Firebase Hosting (issue #42)" is.
+3. Merge.
+
+## Workflow: removing an override (re-converge with canonical)
+
+1. Remove the entry from `.sync-overrides.yml`.
+2. From a Mergepath checkout, run a dry-run sync to confirm the next propagation will write the canonical version:
+   ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)